app/modules/node.c

```diff
@@ -448,6 +449,39 @@ static int node_compile( lua_State* L )
   return 0;
 }
 
+// Task callback handler for node.tasknow()
+static task_handle_t do_node_task_handle;
+static void do_node_task (task_param_t task_fn_ref, uint8_t prio)
+{
+  UNUSED(prio);
+  lua_State* L = lua_getstate();
+  lua_rawgeti(L, LUA_REGISTRYINDEX, (int)task_fn_ref);
+	luaL_unref(L, LUA_REGISTRYINDEX, (int)task_fn_ref);
+  lua_pushinteger(L, prio);
+	lua_call(L, 1, 0);
+}
+
+// Lua: node.tasknow(task_cb)) -- schedule a task for execution next
+static int node_task_post( lua_State* L )
+{
+  int n = 1, Ltype = lua_type(L, 1);
+  unsigned priority = TASK_PRIORITY_MEDIUM;
+  if (Ltype == LUA_TNUMBER) {
+    priority = (unsigned) luaL_checkint(L, 1);
+    luaL_argcheck(L, priority <= TASK_PRIORITY_HIGH, 1, "invalid  priority");
+    Ltype = lua_type(L, ++n);
+  }
+  luaL_argcheck(L, Ltype == LUA_TFUNCTION || Ltype == LUA_TLIGHTFUNCTION, n, "invalid function");
+	lua_pushvalue(L, n);
+
+	int task_fn_ref = luaL_ref(L, LUA_REGISTRYINDEX);
+
+  if (!do_node_task_handle)  // bind the task handle to do_node_task on 1st call
+    do_node_task_handle = task_get_id(do_node_task);
```

Comment:
> In hindsight, should `task_get_id()` really have been called `task_get_handle()` or perhaps even something like `task_create()`?

@jmattsson @pjsg Johny,this would be a trivial fix because at the moment only you, Philip and me use this.  I did toss-up on what to name this, but if either of you feel strongly then make the change.